yaksh/templates/404.html

```diff
@@ -1,5 +1,5 @@
 {% extends "base.html" %}
 
 {% block content %}
-The requested page does not exist.
+{{ exception }}
```

Comment:
<p>Could this exception come with a note like:</p>
It seems that you have encountered an error<br/>
Type of error - {{ exception }} <br/>
Please contact us at python@fossee.in